.github/ISSUE_TEMPLATE/1-regression.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Regression Issue
+about: Use this template for reporting a regression
+---
```

Comment:
How to reproduce it?

Reply:
The column "Describe the current behavior" is for issue reproduce.